packages/manager/components/web-property/newProperty.tsx

```diff
@@ -159,7 +159,7 @@ const NewProperty: FunctionComponent<NewPropertyProps> = ({ webProp }: AnyProps)
       if (!propertyRes?.response?.id) {
         setErrorMessage(propertyRes.response);
         setAlert([
-          { title: `${propertyRes.response}`, variant: 'danger', key: getUniqueId() },
+          { title: `Unable to process the request, Please try again if this problem persists, please contact admin spaship_support@redhat.com`, variant: 'danger', key: getUniqueId() },
```

Comment:
Can you put this string inside a TSX/js  file as a constant and then use it instead of declaring the same string  in multiple places, it will increase the maintainability of the application

Reply:
inside of a  common TSX/js file

Reply:
Sure, we'll work on it separately.